lua/lspconfig/ui/windows.lua

```diff
@@ -102,15 +107,19 @@ function win_float.percentage_range_window(col_range, row_range, options)
   win_opts.col = math.floor(vim.o.columns * col_start_percentage)
   win_opts.width = math.floor(vim.o.columns * width_percentage)
 
-  local bufnr = options.bufnr or vim.api.nvim_create_buf(false, true)
-  local win_id = vim.api.nvim_open_win(bufnr, true, win_opts)
-  vim.api.nvim_win_set_buf(win_id, bufnr)
+  local bufnr = options.bufnr or api.nvim_create_buf(false, true)
+  local win_id = api.nvim_open_win(bufnr, true, win_opts)
+  api.nvim_win_set_option(win_id, 'winhl', 'FloatBorder:LspInfoBorder')
+  api.nvim_win_set_buf(win_id, bufnr)
 
-  vim.cmd 'setlocal nocursorcolumn ts=2 sw=2'
+  api.nvim_win_set_option(win_id, 'cursorcolumn', false)
+  api.nvim_buf_set_option(bufnr, 'tabstop', 2)
+  api.nvim_buf_set_option(bufnr, 'shiftwidth', 2)
```

Comment:
I think it'd be neat to listen to the `VimResized` event and update the window config appropriately:

```lua
api.nvim_create_autocmd({ "VimResized" }, {
  buffer = bufnr,
  callback = function ()
    if api.nvim_win_is_valid(win_id) then
      local updated_win_opts = get_win_opts(options)
      api.nvim_win_set_config(win_id, updated_win_opts)
    end
  end
})
```

I guess this would also have to be applied in `lspinfo.lua` as well, in order to re-render the formatted server list.

Reply:
think about this .when window size changed why we need reset the `ts sw cursorcolumn` or some other options ? Sorry maybe I don't understand what you say :)

Reply:
Here's a recording to illustrate this: https://asciinema.org/a/MMIw0s825KrJeAPqMdxmnOqV4